README.md

```diff
@@ -41,32 +41,21 @@ Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
 
 ## Using Fluent UI React
 
-### Starter apps
+### Create a new app
 
-To create a simple starter app using Fluent UI components, [install the latest LTS node.js](https://nodejs.org), then run:
+To create a simple React app  [install node.js](https://nodejs.org), then run:
 
-```
-npm init uifabric
-```
-
-It'll prompt you for a project name. If you choose to create a project called `my-app`, you can start working on the project like this:
-
-```
+```sh
+npx create-react-app my-app
 cd my-app
 npm start
 ```
 
-This scaffold uses the [`just`](https://github.com/microsoft/just) build library. It is very flexible and requires no "eject" script to allow for customization on its configurations.
-
-If you prefer to use `create-react-app`, [follow these instructions](https://github.com/microsoft/fluentui/wiki/Getting-Started-with-UI-Fabric#create-react-app).
```

Comment:
I think we would want to keep this... there's no harm in linking to our own wiki's creact-react-app page.

Reply:
Almost all of React users and starters use create-react-app, so it drastically simplifies installation instructions in README

Reply:
I think that makes sense as well. There a number of unaddressed considerations here instructing to use the `uifabric` starter app (mainly docs), and those go away by recommending CRA, which has a lot of usage and documentation available.